pkg/generators/openapi_test.go

```diff
@@ -343,6 +343,239 @@ Type: []string{"object"},
 `, funcBuffer.String())
 }
 
+func TestNestedStruct(t *testing.T) {
+	callErr, funcErr, assert, callBuffer, funcBuffer := testOpenAPITypeWriter(t, `
+package foo
+
+// Blah demonstrate a struct with struct field.
```

Comment:
Comment seems wrong, this is not "Blah"

Reply:
There are several such locations. The comment was intended for the complete test referring to the included type `Blah`. In this sense the comment is correct.  But you are completely right. In GO such a comment is always related to the directly following type declaration, which is misleading here. I'll fix this.